### DISCUSSION TITLE
Call to undefined function ScssPhp\ScssPhp\ctype_space()

### DISCUSSION THREAD

**Discussion by AntGS83** [bootscore/bootscore#528] [Q&A]
I've just activated Boostcore on my prod environment (php 8) and i have this error. When i work with my local environment (php 8) i don't have this issue. I don't know what to do so i ask for help here before asking my hosting provider.
Thx for your help !
` Fatal error: Uncaught Error: Call to undefined function ScssPhp\ScssPhp\ctype_space() in /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php:2462 Stack trace: #0 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(2371): ScssPhp\ScssPhp\Parser->expHelper(Array, 0) #1 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(2229): ScssPhp\ScssPhp\Parser->expression(NULL) #2 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(2209): ScssPhp\ScssPhp\Parser->genericList(NULL, 'expression') #3 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(2229): ScssPhp\ScssPhp\Parser->spaceList(NULL) #4 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(2154): ScssPhp\ScssPhp\Parser->genericList(NULL, 'spaceList', ',') #5 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(550): ScssPhp\ScssPhp\Parser->valueList(NULL) #6 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php(281): ScssPhp\ScssPhp\Parser->parseChunk() #7 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Compiler.php(508): ScssPhp\ScssPhp\Parser->parse('@import "bscore...') #8 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scss-compiler.php(59): ScssPhp\ScssPhp\Compiler->compileString('@import "bscore...') #9 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/functions.php(268): bootscore_compile_scss() #10 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/class-wp-hook.php(308): bootscore_scripts('') #11 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array) #12 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/plugin.php(517): WP_Hook->do_action(Array) #13 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/script-loader.php(2194): do_action('wp_enqueue_scri...') #14 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/class-wp-hook.php(308): wp_enqueue_scripts('') #15 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array) #16 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/plugin.php(517): WP_Hook->do_action(Array) #17 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/general-template.php(3049): do_action('wp_head') #18 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-child-main/header.php(31): wp_head() #19 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/template.php(783): require_once('/datas/vol2/w4a...') #20 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/template.php(718): load_template('/datas/vol2/w4a...', true, Array) #21 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/general-template.php(48): locate_template(Array, true, true, Array) #22 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-child-main/front-page.php(11): get_header() #23 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-includes/template-loader.php(106): include('/datas/vol2/w4a...') #24 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-blog-header.php(19): require_once('/datas/vol2/w4a...') #25 /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/index.php(17): require('/datas/vol2/w4a...') #26 {main} thrown in /datas/vol2/w4a149989/var/www/revue-pouvoirs.fr/htdocs/wp-content/themes/bootscore-main/inc/scssphp/src/Parser.php on line 2462`

**Comment by crftwrk** [comment]
Hi,

it seems that the server doesn‘t like the compiler. As you mentioned it works fine in your local enviroment. Please try a different server and/or ask your hoster.

**Reply by AntGS83** [reply]
That was it, i changed the server and it's now ok !
